Improve lottery CLI function (#41)
infomofo/ts-tarot-cards: clio-llm-config.md

- Make sure to note if there are common symbols, suits, numerology, or other significant occurrences across the spread or reading

### Lottery Number Divination Guidelines
When interpreting cards for lottery number selection or when being asked about financial outcomes:

> - Provide a balanced interpretation of the potential for these numbers, considering both positive and negative aspects.

Reviewer: ```suggestion
- Provide a balanced interpretation of the potential for these numbers, considering both positive and negative aspects, including the numerology, symbols of the cards, graphical art of the cards.
```

- Analyze the mystical significance of the cards in relation to fortune, chance, and timing.
- Consider patterns and symbolism that relate to fortune and luck.

> - Avoid making definitive predictions, but offer guidance on the timing and potential of the numbers drawn.

Reviewer: ```suggestion
- Make a definitive statement on if it is a good or idea to play these numbers, offer guidance on the timing and potential of the numbers drawn.
```

> - Emphasize that the cards offer perspective, not a guarantee of wealth.

Reviewer: ```suggestion
```
This is obvious
